### PR TITLE
expanding query checks to VFBconnect

### PR DESCRIPTION
**Pull request by Robbie1977**

### CODE REVIEW

**Comment by Robbie1977** [comment]
I need to tweak the formatting of the new queries

File: src/vfb_query_builder/test/query_roller_tests.py [hunk 1]
```diff
@@ -52,6 +52,53 @@ def get_edits_string(old, new):
                             break
         print('New vfb.xmi')
         print(''.join(xmi))
-                            
+
+        queries = json.loads(term_info_export(escape='json'))
+        # extract out the cypher queries from the VFBconnect master branch
+        ecore=urllib.request.urlopen("https://github.com/VirtualFlyBrain/VFB_connect/raw/master/src/vfb_connect/resources/VFB_TermInfo_queries.json")
+        model = []
+        xmi = []
```
Comment (hkir-dev) [inline src/vfb_query_builder/test/query_roller_tests.py:60]: do we need both xmi and model? Will this do the same job:
```
        for line in ecore:
            xmi.append(line.decode('utf-8'))
        for query in queries:
            print(query)
            print(newquery(queries[query]))
            for index, line in enumerate(xmi):
                if query in line:
                    xmi[index] = queries[query]
                    # break
```

File: src/vfb_query_builder/query_roller.py [hunk 2]
```diff
@@ -626,10 +626,10 @@ def anat_2_ep_query(self, short_forms, *args, pretty_print=False):
                              query_short_forms=short_forms,
                              clauses=[self.anat_2_ep_wrapper(),
                                       aci],
-                             q_name='Get JSON for anat_2_ep query',
+                             q_name=q_name,
                              pretty_print=pretty_print)
 
-    def ep_2_anat_query(self, short_forms, *args, pretty_print=False):
+    def ep_2_anat_query(self, short_forms, *args, pretty_print=False, q_name='Get JSON for anat_2_ep query'):
```
Comment (hkir-dev) [inline src/vfb_query_builder/query_roller.py:632]: q_name should be `Get JSON for ep_2_anat_query`

File: src/vfb_query_builder/test/query_roller_tests.py [hunk 3]
```diff
@@ -52,6 +52,53 @@ def get_edits_string(old, new):
...
+                    for index, line in enumerate(xmi):
+                        if cypher in line:
+                            xmi[index] = line.replace(cypher, queries[query])
+                            break
+        print('New VFB_TermInfo_queries.json')
+        print(''.join(xmi))
+
+        queries = json.loads(single_input_export(escape='json'))
+        # extract out the cypher queries from the VFBconnect master branch
+        ecore=urllib.request.urlopen("https://github.com/VirtualFlyBrain/VFB_connect/raw/master/src/vfb_connect/resources/VFB_results_single_input.json")
```
Comment (hkir-dev) [inline src/vfb_query_builder/test/query_roller_tests.py:81]: function q_name starts with `Get JSON for `, but in VFB_results_single_input.json we don't have this prefix

File: src/vfb_query_builder/test/query_roller_tests.py (hunk 1 above)
Comment (hkir-dev) [reply src/vfb_query_builder/test/query_roller_tests.py:60]: note: seems this won't work if brace brackets are in the same line with the queries as in VFB_results_single_input.json

File: src/vfb_query_builder/query_roller.py (hunk 2 above)
Comment (hkir-dev) [reply src/vfb_query_builder/query_roller.py]: fixed

**Pull request merged**